### ISSUE TITLE
Cookie banner and accessibility

### ISSUE THREAD

**Issue by aschanMarcus** [elleracompany/craft-cookie-consent#52]
The cookie banner should be the first thing in the DOM for keyboard and screen-reader users. While you can achieve this by putting the `before-body-end` hook after the opening body tag, it would make more sense to rename the hook, e.g. `after-body-begin`.
